PowerShellGet/PSModule.psm1

```diff
@@ -1881,11 +2002,13 @@ function Install-Module
 
                 $PSBoundParameters["Name"] = $psgetModuleInfo.Name
                 $PSBoundParameters["RequiredVersion"] = $psgetModuleInfo.Version
+                $PSBoundParameters[$script:AllowPrereleaseVersions] = $psgetModuleInfo.AdditionalMetadata.IsPrerelease -match "true"

```

Comment:
>-match [](start = 119, length = 6)

Any specific reason for using match here? consider using -eq

Reply:
Sure. I thought to use match because I wasn't sure if the string value would be capitalized or not.  But after a little experiment I noticed that both -match and -eq are case insensitive.  I'll change it.

---
In reply to: [140967186](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r140967186) [](ancestors = 140967186)

Reply:
JFYI: -match returns true for ('SomeTrue' -match 'true')

---
In reply to: [141118332](https://github.com/rebro-msft/PowerShellGet/pull/2#discussion_r141118332) [](ancestors = 141118332,140967186)